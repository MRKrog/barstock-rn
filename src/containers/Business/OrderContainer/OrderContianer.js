import React, { Component } from "react";
import { View, Text, StyleSheet } from "react-native";
import Swiper from "react-native-swiper";
import  CurrentOrder from "../CurrentOrder/CurrentOrder";
import  DistributorInventory from "../DistributorInventory/DistributorInventory";

class OrderContainer extends Component{
    constructor(){
        super();
        this.state = {
            test: "test"
        }
    }

    render(){
        return(
            <Swiper index={0} loop={false}>
                <DistributorInventory />
                <CurrentOrder />
            </Swiper>
        )
    }
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        alignItems: "center",
        justifyContent: "center"
    }
})

export default OrderContainer
