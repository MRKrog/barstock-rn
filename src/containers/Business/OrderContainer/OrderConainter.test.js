import { shallow } from "enzyme"
import { OrderContainer } from "./OrderContainer"
import React from "react"

describe("OrderContainer", () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <OrderContainer />
    )
  })

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot()
  })
})