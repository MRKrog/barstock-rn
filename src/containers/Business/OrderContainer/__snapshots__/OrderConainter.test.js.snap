// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderContainer should match snapshot 1`] = `
<_default
  automaticallyAdjustContentInsets={false}
  autoplay={false}
  autoplayDirection={true}
  autoplayTimeout={2.5}
  bounces={false}
  disableNextButton={false}
  horizontal={true}
  index={0}
  loadMinimal={false}
  loadMinimalSize={1}
  loop={false}
  onIndexChanged={[Function]}
  pagingEnabled={true}
  removeClippedSubviews={true}
  scrollsToTop={false}
  showsButtons={false}
  showsHorizontalScrollIndicator={false}
  showsPagination={true}
  showsVerticalScrollIndicator={false}
>
  <Connect(DistributorInventory) />
  <Connect(CurrentOrder) />
</_default>
`;
